SUPPORT.md

```diff
@@ -0,0 +1,5 @@
+# Ark Support
```

Comment:
oh I see---it's a bit of duplication, but it might be worth still putting a link to the mailing lists in `README.md#contributing` **in adddition** to here in `SUPPORT.md`? Just so they know a mailing list exists even if they don't click issues